bats/tests/containers/docker-buildx-python3-uname.bats

```diff
@@ -0,0 +1,55 @@
+load '../helpers/load'
+
+local_setup() {
+    TEMP=/tmp
+    if is_windows; then
+        # We need to use a directory that exists on the Win32 filesystem
+        # so the ctrctl clients can correctly map the bind mounts.
+        # We can use host_path() on these paths because they will exist
+        # both here and in the rancher-desktop distro.
+        TEMP="$(win32env TEMP)"
+    fi
+    WORK_DIR="$TEMP/rd-bats-buildx-dockerDir"
+    RD_CONTAINER_ENGINE=moby
+}
+
+@test 'start' {
+    factory_reset
+    start_container_engine
+    wait_for_container_engine
+}
+
+@test 'create the source directory to work in' {
+    mkdir -p "$WORK_DIR"
+    cd "$WORK_DIR"
+    cat >Dockerfile <<'EOF'
+FROM  registry.access.redhat.com/ubi8/python-39:1-57
+
+RUN  python3 -m pip install  tornado
+
+CMD echo "Running on $(uname -m)"
+EOF
+}
+
+@test 'build the container' {
+    ctrctl buildx create --name amd64builder || true
```

Comment:
I don't know why we should have the `|| true` once we delete the builder in the `start` test.

Also, since we skip the test when using non-docker, we can just write `docker` instead of `ctrctl`. Especially given that `nerdctl buildx` does not exist.
```suggestion
    skip_unless_using_docker
    docker buildx create --name "$BUILDX_BUILDER"
```